### PR TITLE
update dockerfile to support remote builds

### PR DESCRIPTION
**Pull request by based64god**
Also updates the version of cmake and ubuntu that we build against. It might be worth migrating the base image to an nvidia one.

### CODE REVIEW

**Comment by based64god** [comment]
PTAL @Cookt2

File: dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Fletch Dockerfile
-
-FROM ubuntu:trusty
+ARG UBUTNU_VER=18
+FROM ubuntu:${UBUTNU_VER}
```
Comment (Cookt2) [inline dockerfile]: ubutnu -> ubuntu?

**Comment by Cookt2** [comment]
Otherwise looks good to me

File: dockerfile (hunk 1 above)
Comment (based64god) [reply dockerfile]: whoops

**Pull request merged**